MaxText/configs/base.yml

```diff
@@ -392,4 +392,5 @@ enable_single_controller: False
 allow_split_physical_axes: False
 
 # Attributes needed for the MaxText checkpointing loop using standalone_checkpointer.
-gcs_metrics_bucket: distributed-checkpointing-metrics
\ No newline at end of file
+gcs_metrics_bucket: distributed-checkpointing-metrics
+step_time_seconds: 0.0
```

Comment:
What about renaming this to `per_step_interval` to get alignment with the training benchmark? The `per_step_interval` also makes sense as we will be simulating real training step (where each step takes certain times) and writing checkpoint every X steps.

Reply:
Done